libraries/AP_RangeFinder/AP_RangeFinder_Params.cpp

```diff
@@ -12,7 +12,7 @@ const AP_Param::GroupInfo AP_RangeFinder_Params::var_info[] = {
 
     // @Param: PIN
     // @DisplayName: Rangefinder pin
-    // @Description: Analog or PWM input pin that rangefinder is connected to.  Airspeed ports can be used for Analog input, AUXOUT can be used for PWM input
+    // @Description: Analog or PWM input pin that rangefinder is connected to. Airspeed ports can be used for Analog input, AUXOUT can be used for PWM input. When using analog pin 103, the maximum value of the input in 3.3V.
```

Comment:
I don't think we need to change the descriptions. The change is correct, but implies that pin 103 is the only case, that would also be true for all of the aux pins.

Reply:
I haven't had luck reading a voltage on an AUX pin in a script. I think it is import to note that the upper limit range of the parameter changes from pin to pin. That would make it clear when an external voltage divider is necessary to correctly measure the full scale of a sensor. I could add the other pins to the descriptions too.

Reply:
It might be that the aux pins only work for the PWM input not analog, these params are shared between backends I think. Of course the max voltage would still be 3.3v in that case. I tend to think were better off documenting on the wiki or per board readme's. And of course this PR does not actually change the max  voltage spec.

No too fussed tho, happy for it to go in as is.